integration-tests/features/stack_analyses_v2_golang.feature

```diff
@@ -0,0 +1,128 @@
+Feature: Stack analysis v2 API Golang Ecosystem Tests
+
+
+    @sav2
+    Scenario: Check the golang API without user key
```

Comment:
I dont think this is required. The API is is the same. We need not be testing this for all ecosystems.

Reply:
okay